superset-frontend/src/components/Alert/Alert.stories.tsx

```diff
@@ -44,6 +44,7 @@ export const AlertGallery = () => (
           type={type}
           showIcon
           closable
+          closeIcon={<span aria-label="close icon">x</span>} // Custom close icon
```

Comment:
Maybe remove `closeIcon` from here and create a separate Alert component with a custom close icon just to show this option in storybook

Reply:
Thanks for the suggestion! I opted to showcase the closeIcon directly in the existing AlertGallery to keep things simple and avoid adding extra components. Since closeIcon isn't part of the Storybook args, I think this approach still effectively demonstrates its customization without adding another component.